lib/components/Card/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../theme';
+
+import CardTitleGraphic from './CardTitleGraphic.png';
+
+const CardStyled = styled.div`
+  background: ${theme.colors.white};
+  color: ${theme.colors.text};
+
+  border-radius: ${theme.units.borderRadius};
+
+  /* So children don't flow outside border-radius. */
+  overflow: hidden;
+
+  /* So last child's bottom left/right borders don't get cut off. */
+  > :last-child {
+    border-bottom-right-radius: ${theme.units.borderRadius};
+    border-bottom-left-radius: ${theme.units.borderRadius};
+  }
+`;
+
+const CardTitleStyled = styled.div`
+  background: ${theme.colors.primary};
+  color: ${theme.colors.white};
+
+  font-weight: bold;
+  font-size: 18px;
+
+  border: 1px solid ${theme.colors.primary};
+
+  padding: ${theme.units.padding};
+
+  background-image: url(${CardTitleGraphic});
+  background-size: 167px 96px;
+  background-repeat: no-repeat;
+  background-position: top right -50px;
+`;
+
+const CardContentStyled = styled.div`
+  padding: ${theme.units.padding};
+
+  border-bottom: 1px solid ${theme.colors.grayLight};
+  border-left: 1px solid ${theme.colors.grayLight};
+  border-right: 1px solid ${theme.colors.grayLight};
+
+  /*
+    Setting margin-top and margin-bottom so that the spacing set by the above
+    padding is consistent regardless of the first/last element used.
+  */
+  > :first-child {
+    margin-top: 0;
+  }
+
+  > :last-child {
+    margin-bottom: 0;
+  }
+`;
+
+const Card = ({ children, title = false }) => (
+  <CardStyled>
+    {title && <CardTitleStyled>{title}</CardTitleStyled>}
+    {children}
+  </CardStyled>
+);
```

Comment:
nit: I _think_ this would be more useful if the outermost thing here was a styled component, i.e. if it was written like this:

```
export default styled(Card);
```

Per the [caveat in the docs about referring to components](https://styled-components.com/docs/advanced#referring-to-other-components). If we do this (which is similar to the way we do HOC wrapping/composition of most components in Triton), we'll be able to build on perts-ui components later, e.g.

```
const CustomCard = styled.div`
  ${Card} {
    cool: styles;
  }
`;
```

Reply:
d4252f9 Agreed. `export default styled(Card);` isn't allowed, but passed down `className` to allow for styled-components wrapping.

I went ahead `classnames` even though it's not specifically needed now, because I'd rather every component follow the same pattern, and it's all ready to go if we do need to add additional classNames or classNames logic.